whelk-core/src/main/groovy/whelk/Whelk.groovy

```diff
@@ -129,12 +132,12 @@ class Whelk {
             initDocumentNormalizers()
         }
     }
-    
+
+    // FIXME: de-KBV/Libris-ify: some of these are KBV specific, is that a problem?
```

Comment:
Not first priority.
`language` and `identifiedBy` work on KBV terms but there is no big harm in leaving them in here for now

Reply:
Nice to have: refactor and make e.g. `heuristicLinkers` part of core. Depends on extracting/separating "system core" in vocab